lib/upload_system_log.pm

```diff
@@ -44,19 +45,23 @@ sub system_status {
 }
 
 sub journalctl_log {
-    my ($self, $sys_log) = @_;
-    $sys_log //= "/tmp/journalctl.log";
-    script_run("journalctl -b >$sys_log", 40);
+    my ($self, $log) = @_;
+    $log //= "/tmp/journalctl.log";
+    script_run("journalctl -b >$log", 40);
+    return $log;
+}
 
-    return $sys_log;
+sub dmesg_log {
+    my ($self, $log) = @_;
+    $log //= "/tmp/dmesg.log";
+    script_run("dmesg >$log", 40);
+    return $log;
 }
 
 sub upload_system_logs {
-    my $log     = system_status();
-    my $sys_log = journalctl_log();
-
-    upload_logs($log,     timeout => 100);
-    upload_logs($sys_log, timeout => 100);
+    upload_logs(system_status(),  timeout => 100);
```

Comment:
usually, we don't want the test to fail if something goes wrong with log upload, please add `failok => 1` to every upload_logs call

Reply:
But why should it fail? The usual story goes like this: everything works but log status is ignored, log upload starts to fail but nobody notices because status is ignored, then the test fails for real, people are looking for logs and find none because they have not worked for long or try to hunt for the fail root cause in the failed logs steps and get confused

Reply:
To be honest, I can live without `failok => 1` (added on Anton's request), other things are more important :). On the other hand we set it so in LTP , as we had problems with uploading logs, which stopped whole testing (it failed in install_ltp.pm, which is parent of many tests).
@asmorodskyi Keep it or not?

Reply:
I think that test which is checking "feature A" should fail only when "feature A" is broken. Uploading logs should fail only network tests also not all of them. And here we speaking not even about test code but something which suppose to help to analyze test result. Problem should and could be solved using other ways.

Reply:
I would definitely vote to keep it. Scenario which you just describe from LTP clearly prove this to me

Reply:
I vote to keep it too. If test fail by log upload part but pass in what it test, I will probably just rerun whole test, it's a kind of waste osd resources. If the checkpoint fail and log missing we will notice the log missing and fix it.